### ISSUE TITLE
feat(appium): use 64bit emulators

### ISSUE THREAD

**Issue by sjelin** [angular/webdriver-manager#50]
Currently we always use the normal `emulator` executable instead of the various  `emulator64*` executables.

**Comment by dmendes-daitan** [comment]
Any news on that ?

**Comment by sjelin** [comment]
No, and right now we're prioritizing work on selenium 3 so it'll be a bit
